rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala

```diff
@@ -212,6 +212,33 @@ class ProcPrinterSpec extends FlatSpec with Matchers {
     result shouldBe target
   }
 
+  "Receive" should "print peek" in {
```

Comment:
Could you make use of the `checkRoundTrip` function in this test? Makes it much more concise and readable :)

Reply:
True, done